Hi3Helper.Core/Lang/en.json

```diff
@@ -65,51 +65,51 @@
   "_MainPage": {
     "PageTitle": "Root Page",
     "RegionChangeConfirm": "Game region will be changed. Do you want to continue?",
-    "RegionChangeConfirmBtn": "Yes, change it",
+    "RegionChangeConfirmBtn": "Yes, change it!",
     "RegionLoadingTitle": "Loading Region",
     "RegionLoadingAPITitle1": "Loading",
     "RegionLoadingAPITitle2": "Launcher API",
     "RegionLoadingSubtitleTimeOut": "Loading Region {0} has timed-out (> {1} seconds). Retrying...",
-    "RegionLoadingSubtitleTooLong": "It takes a bit longer than expected. Please ensure that your connection is stable.",
+    "RegionLoadingSubtitleTooLong": "Retry is taking a bit longer than expected. Please ensure that your connection is stable!",
     "NavCtrlTitleSettings": "App Settings",
-    "NotifNeverAsk": "Never show me this again",
+    "NotifNeverAsk": "Never show again",
     "MetadataUpdateTitle": "Launcher Metadata Update is Available",
-    "MetadataUpdateSubtitle": "New metadata for the launcher is available! We recommend you to proceed the update in order to ensure that the launcher could communicate with the miHoYo/HoYoverse servers. Click on \"Update Metadata\" to update.",
+    "MetadataUpdateSubtitle": "New metadata for the launcher is available! We recommend you to proceed with installing the update in order to ensure that the launcher can communicate with the miHoYo/HoYoverse servers. Click on \"Update Metadata\" to update.",
     "MetadataUpdateBtn": "Update Metadata",
-    "MetadataUpdateBtnUpdating": "Updating Metadata..."
+    "MetadataUpdateBtnUpdating": "Updating Metadata, please wait..."
   },
 
   "_HomePage": {
     "PageTitle": "Launcher",
     "PreloadTitle": "Pre-load is Available!",
-    "PreloadNotifTitle": "Pre-load package for v{0} is available! Click \"Download\" to downloading it in background.",
-    "PreloadNotifSubtitle": "Pre-load package for v{0} is available! Click \"Download\" to downloading it in background.",
+    "PreloadNotifTitle": "Pre-load package for v{0} is available!",
+    "PreloadNotifSubtitle": "Click \"Download\" to begin downloading it in background.",
     "PreloadNotifCompleteTitle": "Pre-load package has been downloaded!",
     "PreloadNotifCompleteSubtitle": "You have downloaded pre-load package for v{0}!",
-    "PreloadNotifIntegrityCheckBtn": "Check Integrity",
+    "PreloadNotifIntegrityCheckBtn": "Verify Package Integrity",
     "StartBtn": "Start Game",
-    "StartBtnRunning": "Game is Running",
+    "StartBtnRunning": "Game is Running!",
     "VerifyingPkgTitle": "Verifying Package",
-    "PreloadDownloadNotifbarTitle": "Downloading Preload",
+    "PreloadDownloadNotifbarTitle": "Pre-load Download",
```

Comment:
Original title bar give more context as its currently "Downloading" the preload, so I think its fine to leave it as is

Reply:
I do agree that the initial text is more informative, but the reason I changed it was to have it more in line with the `id` of the string. I will revert this though, as the previous string does provide more clarity on the operation being done.

Reply:
I'm agree. We could use ``Downloading Pre-load Package`` instead.

Reply:
Sounds good, will make the changes!